### PR TITLE
Change hosts value in OAM examples

### PR DESCRIPTION
**Pull request by desagar**

### CODE REVIEW

File: oam-application-operator/examples/bobs-books/README.md [hunk 1]
```diff
@@ -97,13 +97,24 @@ Alternatively, you can specify the credentials in environment variables: `GHCR_U
...
+      129.146.194.248 bobbys-books.verrazzano.io
+      129.146.194.248 bobs-orders.verrazzano.io
+      ```
+      a. Open a browser and navigate to the Robert's Books UI at `http://roberts-books.verrazzano.io/`.
+
+      b. Navigate to the Bobby's Books UI at `http://bobbys-books.verrazzano.io/bobbys-front-end`.
+
+      c. Navigate to the Bob's order manager  UI at `http://bobs-orders.verrazzano.io/bobs-bookstore-order-manager/orders`.
+
+    * **Option 2:** Alternatively, you can point your own DNS name to the Load Balancer's external IP address. In this case, you would need to have edited the `bobs-books-app.yaml` file to use the appropriate values under the `hosts` section for each app (such as `application-host.your.domain`, etc.), before deploying the applications.
```
Comment (jmaron99) [inline oam-application-operator/examples/bobs-books/README.md:113]: does option 2 assume that they've been able to register a DNS A/CNAME record for the application in their DNS registry?  I'm not sure I understand....

Comment (desagar) [reply oam-application-operator/examples/bobs-books/README.md:113]: Option 2 means they already have a DNS name, and they are able to add e.g. a subdomain record for the application.

File: oam-application-operator/examples/todo/README.md [hunk 2]
```diff
@@ -97,9 +97,19 @@ To-Do List is a demo application which contains a WebLogic component and an Ingr
    istio-ingressgateway   LoadBalancer   10.96.192.18   129.146.199.184   80:31380/TCP,443:31390/TCP   13d
    ```   
 
-1. Access the To-Do example application in a browser.  This should take you to a page titled 'Derek's To-Do List' with an edit field and an 'add' button that allows you to add things to a to-do list.
-   ```
-   http://129.146.199.184/todo/
-   ```
+1. Access the To-Do example application.
```
Comment (rosemarymarano) [inline oam-application-operator/examples/todo/README.md:100]: Comments from the top of the page:
To-Do List -> ToDo List (please change this globally, see the Lift-and-Shift Guide at https://verrazzano.io/docs/guides/lift-and-shift/). I suggest that the first paragraph needs improvement. If you want to discuss input, let me know.
Also, the Deploy the example application section needs cleanup.
IMO, this README file doesn't look to be in final form. Correct me if I'm wrong, but I think it needs more work before it is reviewed.

**Review by rosemarymarano — COMMENTED** [review]
Edits and suggestions.

File: oam-application-operator/examples/todo/README.md (hunk 2 above)
Comment (desagar) [reply oam-application-operator/examples/todo/README.md]: I updated To-Do -> ToDo and also modified the first paragraph to remove the incomplete/internal information that was there

File: oam-application-operator/examples/bobs-books/README.md [hunk 3]
```diff
@@ -97,13 +97,24 @@ Alternatively, you can specify the credentials in environment variables: `GHCR_U
...
-
-    a. Open a browser and navigate to the Robert's Books UI at `http://129.146.194.248/`.
-
-    b. Navigate to the Bobby's Books UI at `http://129.146.194.248/bobbys-front-end`.
-
-    c. Navigate to the Bob's order manager  UI at `http://129.146.194.248/bobs-bookstore-order-manager/orders`.
+1. Access the applications. To access the applications in a browser, you will need to do one of the following:
+    * **Option 1:** Temporarily modify the `/etc/hosts` file (on Mac or Linux) or `c:\Windows\System32\Drivers\etc\hosts` (on Windows 10), to add entries mapping the hosts used by the applications to the external IP address assigned to your gateway. For example:
+      ```
+      129.146.194.248 roberts-books.verrazzano.io
```
[the remark this answers is not in the document]
Comment (desagar) [reply oam-application-operator/examples/bobs-books/README.md]: Updated to example.com and used `11.22.33.44` as the IP address

File: oam-application-operator/examples/bobs-books/bobs-books-app.yaml [hunk 4]
```diff
@@ -24,7 +24,7 @@ spec:
             spec:
               rules:
                 - hosts:
-                    - "*"
+                    - "roberts-books.verrazzano.io"
```
[the remark this answers is not in the document]
Comment (desagar) [reply oam-application-operator/examples/bobs-books/bobs-books-app.yaml:27]: updated to example.com

File: oam-application-operator/examples/bobs-books/README.md [hunk 5]
```diff
@@ -97,13 +97,24 @@ Alternatively, you can specify the credentials in environment variables: `GHCR_U
...
+      129.146.194.248 bobs-orders.verrazzano.io
+      ```
+      a. Open a browser and navigate to the Robert's Books UI at `http://roberts-books.verrazzano.io/`.
+
+      b. Navigate to the Bobby's Books UI at `http://bobbys-books.verrazzano.io/bobbys-front-end`.
+
+      c. Navigate to the Bob's order manager  UI at `http://bobs-orders.verrazzano.io/bobs-bookstore-order-manager/orders`.
+
+    * **Option 2:** Alternatively, you can point your own DNS name to the Load Balancer's external IP address. In this case, you would need to have edited the `bobs-books-app.yaml` file to use the appropriate values under the `hosts` section for each app (such as `application-host.your.domain`, etc.), before deploying the applications.
+      You can then use a browser to access each of the applications as shown below:
```
[the remark this answers is not in the document]
Comment (desagar) [reply oam-application-operator/examples/bobs-books/README.md]: fixed

File: oam-application-operator/examples/bobs-books/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (desagar) [reply oam-application-operator/examples/bobs-books/README.md]: fixed

File: oam-application-operator/examples/bobs-books/README.md [hunk 6]
```diff
@@ -97,13 +97,24 @@ Alternatively, you can specify the credentials in environment variables: `GHCR_U
...
 
-1. Access the applications. In the following examples, replace the IP address with the external IP address assigned to your gateway.
-
-    a. Open a browser and navigate to the Robert's Books UI at `http://129.146.194.248/`.
-
-    b. Navigate to the Bobby's Books UI at `http://129.146.194.248/bobbys-front-end`.
-
-    c. Navigate to the Bob's order manager  UI at `http://129.146.194.248/bobs-bookstore-order-manager/orders`.
+1. Access the applications. To access the applications in a browser, you will need to do one of the following:
+    * **Option 1:** Temporarily modify the `/etc/hosts` file (on Mac or Linux) or `c:\Windows\System32\Drivers\etc\hosts` (on Windows 10), to add entries mapping the hosts used by the applications to the external IP address assigned to your gateway. For example:
```
[the remark this answers is not in the document]
Comment (desagar) [reply oam-application-operator/examples/bobs-books/README.md]: fixed

**Review by rosemarymarano — DISMISSED** [review]
Everything looks good except that sub-list under Option 2, https://github.com/verrazzano/verrazzano/blob/50aee29becf1083889524812a794504f4bf13877/oam-application-operator/examples/bobs-books/README.md
It might need a line space between each item.

**Comment by desagar** [comment]
> Everything looks good except that sub-list under Option 2, https://github.com/verrazzano/verrazzano/blob/50aee29becf1083889524812a794504f4bf13877/oam-application-operator/examples/bobs-books/README.md
> It might need a line space between each item.

Fixed

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**